### ISSUE TITLE
Stop using reserved identifiers

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#2419]
src/kvirc/kvs/KviKvsCoreSimpleCommands.h has e.g. `__pContext` which starts with `__`

**Comment by IceN9ne** [comment]
I completely agree with this change, just be aware that there are a ton of issues with identifiers in the code. We use `__tr2qs` everywhere in the code, which will need to be changed, but some of the non-obvious changes that people might not expect are our macros like `_REGFNC`, include guards like `_KVI_KVS_CORECALLBACKCOMMANDS_H_`, and enum `_QueryInfo`.

Rules can be found [here](https://en.cppreference.com/w/cpp/language/identifiers).
